src/main/java/team/chisel/ctm/client/util/Dir.java

```diff
@@ -59,7 +58,7 @@
 	
 	private @NonnullType BlockPos[] offsets = new BlockPos[6];
 
-	private Dir(Direction... dirs) {
+	Dir(Direction... dirs) {
```

Comment:
Why was this made package-private?

Reply:
it's an enum, enum constructors are private by default